oracle/README.md

```diff
@@ -378,7 +381,58 @@ See [service_checks.json][12] for a list of service checks provided by this inte
 
 ## Troubleshooting
 
-### JDBC driver (Linux only)
+### Common problems
+
+#### Oracle Native Client
+- If you encounter a `DPY-6000: cannot connect to database`:
+  ```text
+  Failed to connect to Oracle DB, error: DPY-6000: cannot connect to database. Listener refused connection. (Similar to ORA-12660)
+  ```
+ - Ensure Native Network Encryption or Checksumming are not enabled. If they are enabled, you must use thick mode by setting `use_instant_client: true`.
+
+For more information about setting up the Oracle Instant Client, see the [Oracle integration documentation][3].
+
+#### Oracle Instant Client
+- Verify that both the Oracle Instant Client and SDK files are located in the same directory.
```

Comment:
I see that in the `python-oracledb` [docs](https://python-oracledb.readthedocs.io/en/latest/user_guide/initialization.html#setting-the-oracle-client-library-directory), we have different directions to initialize a Thick mode connection on different OS's. Would it be a good idea to add an optional config option so users can specify their instant client location?